adapter/src/hermes/adapter/interceptor.cc

```diff
@@ -0,0 +1,95 @@
+/**
+ * Internal headers
+ */
+#include "interceptor.h"
+
+/**
+ * Define Interceptor list for adapter.
+ */
+hermes::adapter::InterceptorList list;
+
+namespace hermes::adapter {
+/**
+ * is exit by program called.
+ */
+bool exit = false;
+}  // namespace hermes::adapter
+
+void PopulateBufferingPath() {
+  char* hermes_config = getenv(kHermesConf);
+  hermes::Config config = {};
+  const size_t kConfigMemorySize = KILOBYTES(16);
+  hermes::u8 config_memory[kConfigMemorySize];
+  if (hermes_config && strlen(hermes_config) > 0) {
+    list.hermes_paths_exclusion.push_back(hermes_config);
+    hermes::Arena config_arena = {};
+    hermes::InitArena(&config_arena, kConfigMemorySize, config_memory);
+    hermes::ParseConfig(&config_arena, hermes_config, &config);
+  } else {
+    InitDefaultConfig(&config);
+  }
+  if (config.mount_points->empty()) {
```

Comment:
Why is this check needed? If `config.mount_points` looks like this: `["./nvme", "./burst_buffer"]`, then these paths won't get added to your exclusion list, right?

Reply:
oops. I intended the not.